src/components/apiDelegation/reusables/OverviewPageContent/OverviewPageContent.tsx

```diff
@@ -152,9 +152,14 @@ export const OverviewPageContent = ({
             </Button>
           </div>
         )}
-        <Panel title={t('api_delegation.card_title')}>
-          {t('api_delegation.api_panel_content')}
-        </Panel>
+        <div className={classes.infoPanelWrapper}>
+          <Panel
+            title={t('api_delegation.card_title')}
+            forceMobileLayout
+          >
+            {t('api_delegation.api_panel_content')}
+          </Panel>
+        </div>
         <div className={classes.pageContentContainer}>
           <h2 className={classes.apiSubheading}>{accessesHeader}</h2>
```

Comment:
![image](https://user-images.githubusercontent.com/22012997/216377823-9ab8fb4a-2d15-448b-9bb9-4ef518f1a958.png)

Oh no! The header got smooched! 😮
Luckily, the design has the solution!

Move it up and over the edit-button like depicted under and it'll probably be a lot happier 😊
![image](https://user-images.githubusercontent.com/22012997/216377867-00d0a25e-be15-40f6-8b13-1be455c6c3b6.png)